comment-crop/all-comments.txt

```diff
@@ -0,0 +1,451 @@
+/* @product Called by the current block producer to add block reward to a validator
```

Comment:
I suggest keeping the cropped file within the repository. This will allow for very easy diff-ing of changes

Reply:
and of course, no need to review the 'all-comments' file